### DISCUSSION TITLE
Making wgpu use vulkan instead of dx12

### DISCUSSION THREAD

**Discussion by thunderstorm010** [hecrj/iced#1289] [Q&A]
How to make this? I've tried setting a dependency on wgpu-hal and setting the vulkan feature flag:
```toml
wgpu-hal = { version = "*", features = ["vulkan"] }
```
This results in:
```
error: process didn't exit successfully: `target\debug\app.exe` (exit code: 0xc000041d)
```

**Comment by hecrj** [comment]
Try setting the `WGPU_BACKEND` environment variable to `vulkan` when running your application.
